### PR TITLE
Add Cloudflare Pages link as PR comment

### PR DESCRIPTION
**Pull request by vipulgupta2048**
It's annoying to find the CF pages link from inside the GHA run summary, lost of people have had this friction. 
No more. This PR adds that link to the comments. 

Screenshot 2024-07-11 at 4 18 26 PM

Zulip: Started from https://balena.zulipchat.com/#narrow/stream/408336-aspect.2Fuser-experience/topic/Where.20should.20SAML.20docs.20fit.20in.3F

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
Completed: https://github.com/product-os/flowzone/pull/1109#issuecomment-2221419322

**Comment by vipulgupta2048** [comment]
Completed: https://github.com/product-os/flowzone/pull/1109#issuecomment-2221419322

**Review by ab77 — CHANGES_REQUESTED** [review]
@vipulgupta2048 could you please just add a test so that we know the new step is working in the PR draft/sync stage (before merging). I think flowzone has a CF site already..

**Comment by klutchell** [comment]
I see the test comment in this PR, it just got buried in the history. LGTM

File: flowzone.yml [hunk 1]
```diff
@@ -3283,13 +3283,30 @@ jobs:
...
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
           accountId: ${{ secrets.CF_ACCOUNT_ID }}
           wranglerVersion: "3.5.1" # latest - https://www.npmjs.com/package/wrangler
-          command: --version
-          postCommands: npx wrangler pages deploy --branch ${{ env.CF_BRANCH }} --project-name=${{ inputs.cloudflare_website }} build/ | tee -a $GITHUB_STEP_SUMMARY
+          command: pages deploy --branch ${{ env.CF_BRANCH }} --project-name=${{ inputs.cloudflare_website }} build/
+
+      - name: Find Cloudflare Pages link comment
+        uses: peter-evans/find-comment@v3
```
Comment (klutchell) [inline flowzone.yml:3295]: Suggest pinning to the full semver here

File: flowzone.yml [hunk 2]
```diff
@@ -3283,13 +3283,30 @@ jobs:
...
+        id: find_cf_comment
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body-includes: Website deployed to CF Pages, 👀 preview link
+
+      - name: Create or update Cloudflare Pages link comment if deployment link present
+        uses: peter-evans/create-or-update-comment@v4
```
Comment (klutchell) [inline flowzone.yml:3302]: Suggest pinning to the full semver here

**Review by ab77 — APPROVED** [review]

**Pull request merged**